frontend/app/views/spree/products/_cart_form.html.erb

```diff
@@ -14,17 +14,17 @@
   <%= render 'cart_form_availability_templates', type: 'out-of-stock' %>
 </template>
 
+<% is_product_available_in_currency = !(@product.price_in(current_currency).amount.nil? || @product.price_in(current_currency).amount.zero?) %>
```

Comment:
Shouldn't `is_product_available_in_currency` be a method in products_helper rather than assigning it on a template?